obp-commons/src/main/scala/com/openbankproject/commons/dto/JsonsTransfer.scala

```diff
@@ -628,8 +628,7 @@ case class OutBoundUpdateBankAccount(outboundAdapterCallContext: OutboundAdapter
                               accountType: String,
                               accountLabel: String,
                               branchId: String,
-                              accountRoutingScheme: String,
-                              accountRoutingAddress: String
+                              accountRoutings: List[AccountRouting]
```

Comment:
When modifying `obp-commons` any type, need bump up pom version:
change 3 pom.xml files's version from `1.6.3` to `1.7.0`